qml/pages/AboutPage.qml

```diff
@@ -0,0 +1,160 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+import '../components'
+
+Page {
+    id: aboutpage
+
+    readonly property string _APP_VERSION: appVersion
+    readonly property string _APP_BUILD_NUMBER: appBuildNum
+
+    SilicaFlickable {
+        anchors.fill: parent
+        contentHeight: column.height
+
+        Column {
+            id: column
+            width: parent.width
+
+            PageHeader {
+                title: qsTr("About")
+            }
+
+            Image {
+                id: icon
+                source: "image://theme/harbour-nayttamo"
+                anchors.left: parent.left
+                anchors.leftMargin: (aboutpage.width - icon.width) / 2
+            }
+
+            HorizontalSeparator { }
+
+            Label {
+                id: appname
+                text: "Näyttämö"
```

Comment:
This could still have qsTr although we currently don't translate it. Other places have it for app name.

Reply:
done :+1: